src/chttpd/src/chttpd_node.erl

```diff
@@ -0,0 +1,272 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License.  You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+-module(chttpd_node).
+-compile(tuple_calls).
+
+-export([
+    handle_node_req/1,
+    get_stats/0
+]).
+
+-include_lib("couch/include/couch_db.hrl").
+
+-import(chttpd,
```

Comment:
Imports are the devil.

Reply:
they are but it's a house style for chttpd modules given how often we use those functions. 

_config is justified as it has logic around white/blacklisting not present in the couch_httpd one, and _stats has simply moved from chttpd_misc

Reply:
Legacy style mebbe. I don't care enough to ask for a change though.

Ah, good call on the blacklisting. I figured there was enough of a difference that just deleting the specializations wasn't enough.